renderer/src/components/WalletTransactionsHistory.tsx

```diff
@@ -4,23 +4,29 @@ import dayjs from 'dayjs'
 import { ReactComponent as IncomeIcon } from '../assets/img/icons/income.svg'
 import { ReactComponent as OutcomeIcon } from '../assets/img/icons/outcome.svg'
 import { ReactComponent as ExternalLinkIcon } from '../assets/img/icons/external.svg'
+import { ReactComponent as LoadingIcon } from '../assets/img/icons/loading.svg'
 import WalletTransactoinStatusWidget from './WalletTransactionStatusWidget'
 import { brownseTransactionTracker } from '../lib/station-config'
 import WalletOnboarding from './WalletOnboarding'
 
 interface WalletTransactionsHistoryProps {
-  allTransactions: FILTransaction[] | [],
+  allTransactions: FILTransaction[] | [] | null,
```

Comment:
```suggestion
  allTransactions: FILTransaction[] | null,
```

Looks like the empty array doesn't provide any value as a type here

Reply:
ok, I just reviewed since you requested a review 👋